www/class/centreonGraph.class.php

```diff
@@ -206,6 +207,22 @@ public function __construct($user_id, $index = null, $debug = 0, $compress = nul
         $DBRESULT->closeCursor();
         unset($opt);
 
+        /* Get RRDCacheD options */
+        $result = $this->DB->query("SELECT config_key, config_value
+            FROM cfg_centreonbroker_info AS cbi
+            INNER JOIN cfg_centreonbroker AS cb ON (cb.config_id = cbi.config_id)
+            INNER JOIN nagios_server AS ns ON (ns.id = cb.ns_nagios_server)
+            WHERE ns.localhost = '1'
+            AND cbi.config_key IN ('rrd_cached_option', 'rrd_cached')"
+        );
+        while ($row = $result->fetch()) {
+            if ($row['config_key'] === 'rrd_cached_option') {
```

Comment:
could avoid if/elsif : 
$this->rrdCachedOptions[$row['config_key']] = $row['config_value'];